register/views.py

```diff
@@ -41,13 +42,18 @@ def get_success_url(self):
 
 
 class EmailPrefilledMixin:
+    def dispatch(self, request, *args, **kwargs):
+        self.email = self.request.session.get('email')
+        if not self.email:
+            return HttpResponseRedirect(reverse('auth'))
```

Comment:
These URL should never be called from a 3rd service nor a validation. Therefore should never have a _next_ parameter.
The edge case I met was re-posting after a server restart.
The purpose here is to re-enter the email if one of these edgecase were met.